src/hooks/useToast.ts

```diff
@@ -0,0 +1,45 @@
+import { useToast as useToastFromRnToastNotifications } from 'react-native-toast-notifications'
+import { ToastColorType, ToastIconType } from 'screens/ToastView'
+
+const useToast = (): {
+  show: (
+    message: string | JSX.Element,
+    data: { color: ToastColorType; icon?: ToastIconType }
+  ) => string
+  update: (
+    id: string,
+    message: string | JSX.Element,
+    data: { color: ToastColorType; icon?: ToastIconType }
+  ) => void
+  hide: (id: string) => void
+  hideAll: () => void
+} => {
```

Comment:
if we are going to use this, we use replace all toast (everywhere where we use useToast from sendbird lib) with this one

Reply:
I will change